README.md

```diff
@@ -110,189 +192,34 @@ Since we're done with this, let's delete our API endpoint we are going to assign
-```
+## Create some VMs!
 
-Validate the events are being received (this assumes you have kubectl access):
+We are now ready to create VMs.  If everything is correct creating a vm on GCP is a simple as:
 
 ```
-dispatch log event-driver-vcenter -f driver
-...
-time="2018-05-25T20:13:19Z" level=debug msg="got event of type UserLoginSessionEvent"
-time="2018-05-25T20:13:19Z" level=debug msg="No dedicated handler"
-time="2018-05-25T20:13:20Z" level=debug msg="got event of type UserLogoutSessionEvent"
-time="2018-05-25T20:13:20Z" level=debug msg="No dedicated handler"
+/cloudmaster create vmtest01 on gcp
 ```
 
-## Subscribe to vCenter Events
+![success](success.png)
 
-The next step is to create the function which will subscribe to the events from the driver. Create a `status` function
-which will update a slack channel with vCenter events.  Note that we are passing in the `slack` secret which contains
-the status URL pointing to the incoming webhook define at the start:
+## What's next?
 
-```
-dispatch create function status status.py --image python3-pyvmomi --secret slack
-```
+This should be a good introduction into a chat-ops workflow.  This example could be extended in a lot of different
+ways to add new commands and functionality.
 
-Finally, we can subscrive our function to the incoming events:
 
-```
-dispatch create subscription status --source-type vcenter --event-type vm.deployed
-dispatch create subscription status --source-type vcenter --event-type vm.stopping
-dispatch create subscription status --source-type vcenter --event-type vm.removed
-dispatch create subscription status --source-type vcenter --event-type vm.powered.on
-dispatch create subscription status --source-type vcenter --event-type vm.powered.off
-```
+### Appendix: Using Let's encrypt with Dispatch
 
-Your subscriptions should look similar to:
+If you would like to use properly signed certificate with Dispatch, you can do that using `certbot`. Install certbot
+using [official instructions](https://certbot.eff.org/docs/install.html), and then run:
 
 ```
-$ dispatch get subscriptions
-           NAME          | SOURCE TYPE |   EVENT TYPE   | FUNCTION NAME | STATUS |         CREATED DATE
-----------------------------------------------------------------------------------------------------------
-  native-troll-352185    | vcenter     | vm.powered.on  | status        | READY  | Sat Jan  1 12:48:45 PST 0000
-  willing-cheetah-356512 | vcenter     | vm.removed     | status        | READY  | Sat Jan  1 12:48:44 PST 0000
-  touched-foal-822640    | vcenter     | vm.powered.off | status        | READY  | Sat Jan  1 12:48:46 PST 0000
-  just-wildcat-236741    | vcenter     | vm.stopping    | status        | READY  | Sat Jan  1 12:48:44 PST 0000
-  star-goose-402131      | vcenter     | vm.deployed    | status        | READY  | Sat Jan  1 12:48:39 PST 0000
+certbot certonly --standalone -d ato-api.dispatchframework.io
```

Comment:
maybe we should use more generic DNS names `api.example.com` for instance

Reply:
changed to generic DNS name